geopy/util.py

```diff
@@ -80,3 +80,10 @@ def decode_page(page):
         else: # requests?
             encoding = page.headers.get("charset") or "utf-8"
             return str(page.content, encoding=encoding)
+
+
+def get_version():
+    import geopy
+    return str(geopy.__version__)
```

Comment:
`geopy.__init__.__version__` can move to `geopy.version.GEOPY_VERSION` or such to avoid circular imports, with it reimported in the package init for export, yeah?